server/src/test/java/org/elasticsearch/rest/action/admin/cluster/RestAddVotingConfigExclusionActionTests.java

```diff
@@ -56,4 +59,47 @@ public void testResolveVotingConfigExclusionsRequestNodeNames() {
         assertArrayEquals(expected, addVotingConfigExclusionsRequest.getNodeNames());
     }
 
+    public void testResolveVotingConfigExclusionsRequestTimeout() {
+        Map<String, String> params = new HashMap<>();
+        params.put("node_names", "node-1,node-2,node-3");
+        params.put("timeout", "60s");
+        RestRequest request = new FakeRestRequest.Builder(xContentRegistry()).withMethod(RestRequest.Method.PUT)
```

Comment:
It would feel more intuitive to use `POST`, but I also see how it does not matter.

Reply:
Good point, didn't spot that. Must have been left like that from a much earlier iteration.